example/src/utils/MethodUtil.ts

```diff
@@ -190,3 +190,93 @@ export const testSignTransaction: () => Promise<FormattedRpcResponse> = async (
     result: signedTx,
   };
 };
+
+export const readContract = async (
+  web3Provider?: ethers.providers.Web3Provider
+): Promise<FormattedRpcResponse> => {
+  if (!web3Provider) {
+    throw new Error('web3Provider not connected');
+  }
+
+  const daiAddress = 'dai.tokens.ethers.eth';
```

Comment:
In effort to make examples cleaner / less verbose, maybe we can move adresses and abis outside somewhere like `data` folder maybe?

Reply:
yes, i have a constants folder. I'll move them there